.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 **/*.rs.bk
+src/asset.in
```

Comment:
have you pulled in recent changes? this file is now in `target`, so we don't need this line